core/generate/command_step_builder.go

```diff
@@ -15,7 +17,9 @@ type CommandStepBuilder struct {
 	Assets      map[string]string
 	Variables   map[string]string
 	Caches      []string
-	UseSecrets  bool
+	Secrets     *[]string
```

Comment:
Why not just `Secrets     []string`?

Reply:
Because I want to default to all secrets if this is nil when deserialized from a `railpack.json` file. If it is `[]string` then it will deserialize into an empty array

Reply:
Ah, yeah that's legit.